.travis.yml

```diff
@@ -26,15 +25,15 @@ matrix:
 
 before_install:
   - npm config set spin false
-  - npm install -g bower phantomjs-prebuilt
-  - bower --version
+  - npm install -g phantomjs-prebuilt
   - phantomjs --version
-
-install:
+  - npm install -g npm@4
+  - npm --version
+  - npm install -g phantomjs-prebuilt
   - npm install
-  - bower install
 
 script:
-  # Usually, it's ok to finish the test scenario without reverting
   #  to the addon's original dependency state, skipping "cleanup".
   - node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO test --skip-cleanup
+
+  - node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO --skip-cleanup
```

Comment:
I think you should delete the line above this one that does almost the same thing.

Reply:
:+1: